corehq/apps/formplayer_api/smsforms/api.py

```diff
@@ -299,8 +299,6 @@ def formplayer_post_data_helper(d, auth, content_type, url):
     headers = {}
     headers["Content-Type"] = content_type
     headers["content-length"] = len(data)
-    # Remove Cookie header once formplayer supports mac digest header for auth
-    headers["Cookie"] = 'sessionid=%s' % settings.FORMPLAYER_INTERNAL_AUTH_KEY
```

Comment:
@wpride when i tested locally removing this header, formplayer now returns an error saying "Invalid session id". if i put the header back and just pass in any arbitrary value for `sessionid`, i still get an error from formplayer but with an empty exception in the json output